### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by celfro**

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -9,10 +9,37 @@
...
+    :password => params[:password])
+    redirect '/user/login' 
+  end
+
+  get '/user/login' do
+    erb :login
+  end
+
+  post '/user/login' do
+    user = User.first(:email => params[:emailaddress])
```
Comment (ajosephides) [inline app/app.rb]: considering email is unique a User.get would work here. Reading it with a first gives the impression that there are potentially more than one.

File: app/app.rb [hunk 2]
```diff
@@ -9,10 +9,37 @@
 
 class MakersBnB < Sinatra::Base
 
+  enable :sessions
```
Comment (petraartep) [inline app/app.rb]: get rid of the sessions please

File: coverage/.last_run.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (ajosephides) [inline coverage/.last_run.json]: latest pull request will include coverage in the .gitignore

File: lib/booking.rb [hunk 4]
```diff
@@ -4,11 +4,10 @@
 
 class Booking
   include DataMapper::Resource
-  
   property :id, Serial
   property :date, DateTime
   property :status, String
-  
+
```
Comment (ajosephides) [inline lib/booking.rb]: I think in a future change we should include the created_at info

File: lib/user.rb [hunk 5]
```diff
@@ -6,9 +6,9 @@ class User
   include DataMapper::Resource
 
   property :id, Serial
+  property :name, String
   property :email, String, :unique => true
   property :password, String
-  property :name, String
```
Comment (ajosephides) [inline lib/user.rb]: I think in a future change we should include the created_at info

File: spec/user_spec.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+
+
+describe User do
```
Comment (petraartep) [inline spec/user_spec.rb]: please use capybara syntax in the tests, change these to feature and scenario

File: app/app.rb (hunk 1 above)
Comment (LucyMHall) [reply app/app.rb]: Will do :)

File: lib/booking.rb (hunk 4 above)
Comment (LucyMHall) [reply lib/booking.rb]: Ok, i think for the MVP this isn't necessary but it is definitely something to add in later if we get time.

File: app/app.rb (hunk 2 above)
Comment (LucyMHall) [reply app/app.rb]: Will do!

File: spec/user_spec.rb (hunk 6 above)
Comment (LucyMHall) [reply spec/user_spec.rb]: This doesn't make any difference to how it runs, but I agree it is clearer your way

File: app/app.rb (hunk 1 above)
Comment (ajosephides) [reply app/app.rb]: Following discussion - my bad with using get. Sorry - seems docs say use `first`

**Pull request merged**